app/models/course/material/folder/download.rb

```diff
@@ -0,0 +1,7 @@
+class Course::Material::Folder::Download < ActiveRecord::Base
```

Comment:
Given that S3 currently does not authorize the users before the file is downloaded, implementing authorization for the download job at this point does not provide any additional security. Using UUID as the identifier to the file and setting a reasonable expiry period (< a day or shorter) for the download link/file can be a mitigating control for the time being.

Moving forward, to ensure the security requirements are applied consistently in Coursemologyv2’s features, we should implement access control for file downloads in S3 and in the jobs.